src/grammar/query.rs

```diff
@@ -1067,16 +1744,14 @@ Root@0..73
             Ident@15..17 "c3"
       Comma@17..18 ","
       Whitespace@18..19 " "
-      ColumnExpr@19..33
-        FunctionInvocation@19..32
-          IdentGroup@19..28
-            Ident@19..28 "aggregate"
-          LParen@28..29 "("
-          ArgumentList@29..31
-            Argument@29..31
-              IdentGroup@29..31
-                Ident@29..31 "c4"
-          RParen@31..32 ")"
+      ColumnExpr@19..28
+        IdentGroup@19..28
+          Ident@19..28 "aggregate"
+      ColumnExpr@28..33
+        LParen@28..29 "("
+        IdentGroup@29..31
+          Ident@29..31 "c4"
+        RParen@31..32 ")"
```

Comment:
This is no longer the correct AST, as  `aggregate(c4)` should be a function invocation instead of two separate column expressions.